rs/tests/driver/src/driver/ic.rs

```diff
@@ -272,6 +290,10 @@ impl InternetComputer {
             let sks = NodeSecretKeyStore::new(tempdir.join(format!("node-{:p}", n)))?;
             n.secret_key_store = Some(sks);
         }
+        for n in self.api_boundary_nodes.iter_mut() {
```

Comment:
Here you could use the same simplification has [above](https://github.com/dfinity/ic/pull/2290#discussion_r1820939276) and just have a single for-loop iterating over all nodes.

Reply:
Thanks for pointing it out! I updated it.